crates/query-engine/translation/src/translation/query/values.rs

```diff
@@ -176,6 +183,74 @@ pub fn translate_projected_variable(
 
             sql::ast::Expression::CorrelatedSubSelect(Box::new(result_select))
         }
+        database::Type::RangeType(range_type) => {
```

Comment:
Ahh OK, so these are extractors that use JSON paths to reach in and get the parts we need.